source/common/json/config_schemas.cc

```diff
@@ -1539,6 +1539,9 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
         "minimum" : 0,
         "exclusiveMinimum" : true
       },
+      "respect_dns_ttl" : {
```

Comment:
You can revert this change, we no longer make changes to the v1 code, same in cds_json.cc.